client-misk/src/main/kotlin/app/cash/backfila/client/misk/spi/BackfillBackend.kt

```diff
@@ -0,0 +1,10 @@
+package app.cash.backfila.client.misk.spi
+
+/**
+ * Service provider interface for backends like Hibernate and DynamoDb. Backends construct operators
+ * that actually run the backfill.
+ */
+interface BackfillBackend {
```

Comment:
this interface is great

Reply:
Glad you like it! Kudos goes to @swankjesse too.